packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -312,22 +392,59 @@ class ReorderableListView extends StatefulWidget {
   ///  hovering, and [SystemMouseCursors.grabbing] when dragging.
   final MouseCursor? mouseCursor;
 
+  /// The builder for separator widgets if this is a [ReorderableListView.separated].
```

Comment:
Here and elsewhere `...a [ReorderableListView.separated].` would be better stated as when using the [ReorderableListView.separated] constructor.